lib/features/contextPad/ImprovedContextPadProvider.js

```diff
@@ -176,6 +176,10 @@ ImprovedContextPadProvider.$inject = [ 'contextPad', 'eventBus', 'popupMenu' ];
 
 // helpers //////////
 function getIcon(icon) {
+  if (icon === 'connect') {
```

Comment:
Let us not hard-code `draggable` here (based on the `connect` icon), but instead infer whether the icon is draggable from handlers registered with [a particular entry](https://github.com/camunda/improved-canvas/blob/a2b1d352b6ccd69dffb8c955221e76f67a1dbbec/lib/features/contextPad/ImprovedContextPadProvider.js#L103).

This way we know that we _always_ will render the appropriate HTML based on whether a drag handler is registered on the entry or not.